package.json

```diff
@@ -68,7 +68,8 @@
     "node-yaml": "^4.0.1",
     "reflect-metadata": "^0.1.13",
     "ts-node-register": "^1.0.0",
-    "typedi": "^0.10.0"
+    "typedi": "^0.10.0",
+    "uuid": "^10.0.0"
```

Comment:
Node ships a `randomUUID` method via the `crypto` module - are we able to use that and reduce an extra dependency and dev dep?

Reply:
Was using uuid just for validation so we didn't need to include manual regexs. Happy for alternatives though

Reply:
No problem, if we are using it to validate an already existing value, then this is fine 👍